### ISSUE TITLE
Invalid service account specification

### ISSUE THREAD

**Issue by jaeyson** [kreait/firebase-php#451]
**Environment:**
 - Ubuntu 19.10
 - PHP version: php:7.4.7-fpm-alpine (docker)
 - `firebase-php` Version: ^5.6

**Problem:**
I had the same issue with #374 except I used the service account json. Here's how I used it:

```php
<?php //src/FirebaseAdminService.php
namespace Src;

use Kreait\Firebase\Factory;
//use Kreait\Firebase\Firestore;

class FirebaseAdminService
{

    public function __construct()
    {
        $factory = (new Factory)
                          ->withServiceAccount(__DIR__ .'/firebase_credentials.json');
                          // ->withDatabaseUri('https://vuejs-firestoredb.firebaseio.com');

        $firestore = $factory->createFirestore();

        $db = $firestore->database();
    }

  public function checkName()
  {
    $result = $this->db->collection('names');

    echo json_encode([
        'response' => $result
    ]);
  }
}
```

```php 
<?php //public/index.php
require "../vendor/autoload.php";
use Src\FirebaseAdminService as Firebase;

//...

$firebase = new Firebase;
if ($requestMethod == 'GET') {
    $firebase->checkName();
}
```

and it returns this:

```
Uncaught Kreait\Firebase\Exception\InvalidArgumentException:
Invalid service account specification in /var/www/html/vendor/kreait/firebase-php/src/Firebase/ServiceAccount.php:70

Stack trace:
#0 /var/www/html/vendor/kreait/firebase-php/src/Firebase/Factory.php(124):
     Kreait\Firebase\ServiceAccount::fromValue('/var/www/html/s...')

#1 /var/www/html/src/FirebaseAdminService.php(20):
     Kreait\Firebase\Factory-&gt;withServiceAccount('/var/www/html/s...')

#2 /var/www/html/public/index.php(38):
     Src\FirebaseAdminService-&gt;__construct()

#3 {main} thrown in /var/www/html/vendor/kreait/firebase-php/src/Firebase/ServiceAccount.php on line 70
```

**Comment by jeromegamez** [comment]
I just released 5.7.0 which should provide more information on why the Service Account is considered invalid. Please update and let me know if it helps 🤞

**Comment by jaeyson** [comment]
@jeromegamez Thank you, I've used your laravel-firebase package instead just to get my project up and going. Ill definitely check out that version once my project is done. can I close this?
